lgc/tool/lgc/lgc.cpp

```diff
@@ -352,7 +352,10 @@ int main(int argc, char **argv) {
       // Set the triple and data layout, so you can write tests without bothering to specify them.
       TargetMachine *targetMachine = lgcContext->getTargetMachine();
       module->setTargetTriple(targetMachine->getTargetTriple().getTriple());
-      module->setDataLayout(targetMachine->createDataLayout());
+      std::string dataLayoutStr = targetMachine->createDataLayout().getStringRepresentation();
+      // continuation stack address space.
+      dataLayoutStr += "-p32:32:32";
```

Comment:
Is this future-proof?

Reply:
I think we will less likely to change that. But feel free to give your concern here.

Reply:
I just made small update to format the string from cps::stackAddrSpace.